packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -31,10 +31,6 @@ export interface AtomicSearchBoxOptions {
 export class AtomicSearchBox implements AtomicSearchBoxOptions {
   @Element() host!: HTMLDivElement;
   @State() controllerState!: SearchBoxState;
-  @State() strings = {
```

Comment:
This will still be useful for labels, titles, etc. But for displaying strings, it's more simple to reuse the atomic-text component. Customer will also be able to use it anywhere in their search interface